### DISCUSSION TITLE
Dependency on Java 8

### DISCUSSION THREAD

**Discussion by isaac-patterson-xero** [batect/batect#1389] [General]
Hey @charleskorn in the [docs](https://batect.dev/docs/getting-started/requirements#linux-and-macos) it says the dependency on Java 8 will be removed before Batect 1.0. Do you know roughly when this would be?

Cheers, love the tool!

**Comment by charleskorn** [comment]
Hey @isaac-patterson-xero, thanks for the kind words! 

At this point, I don't have a timeframe to share for removing the dependency on the JVM. It's still a goal, but isn't the highest priority right now.
